### PR TITLE
Feature/content tabs

### PR DESCRIPTION
**Pull request by rhelling**
#### Description
Adds initial tabs component

### CODE REVIEW

File: src/stories/molecules/tabs/tabs.stories.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<section class="stories__molecules">
```
Comment (zwaardje) [inline src/stories/molecules/tabs/tabs.stories.html]: extract the content to json, or is this part of the wip

File: src/js/components/molecules/tab/tab-models.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Vue } from 'vue-property-decorator';
+
+export interface ITab extends Vue {
+    title: string;
+    setVisible(state:boolean):void;
```
Comment (zwaardje) [inline src/js/components/molecules/tab/tab-models.ts]: No content?

Comment (rhelling) [reply src/js/components/molecules/tab/tab-models.ts]: Content can be entered into the slot 👍

File: src/stories/molecules/tabs/tabs.stories.html (hunk 1 above)
Comment (rhelling) [reply src/stories/molecules/tabs/tabs.stories.html]: Nice catch, you are right ;-)

Comment (rhelling) [reply src/stories/molecules/tabs/tabs.stories.html]: Didn't really realize yesterday that the content can only be added through the default component slot. So there is no option use json as input.

**Pull request merged**